src/components/utils/popover.ts

```diff
@@ -155,6 +202,16 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
    */
   private scrollLocker = new ScrollLocker()
 
+  /**
+   * Editor container element
+   */
+  private editorElement: HTMLElement;
+
+  /**
+   * Reference to popover item that was clicked but requires second click to confirm action
+   */
+  private itemRequiringConfirmation = null;
```

Comment:
Does it mean that we support only the singe confirmation button at the popover? Could we support several buttons in that state at once? Or how to handle that case?

Reply:
only one popover item at once can be in confirmation state. Once any other element is clicked, confirmation state is disappearing.